validator/src/main/resources/schema/mathml3/mathml3-common.rng

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+      This is the Mathematical Markup Language (MathML) 3.0, an XML
```

Comment:
probably we need to add some copyright blurb to the NOTICE file, it already has something for MathML 1 and 2